src/ui/app/index.tsx

```diff
@@ -31,22 +31,23 @@ const App = () => {
   let routesContent: React.ReactElement;
   routesContent = (
     <Routes>
-      <Route path={`${ pathPrefix ?? "/" }`} element={<RequireAuth user={user}><HomePage user={user} /> </RequireAuth>} />
+      <Route path={`${pathPrefix ?? "/"}`} element={<RequireAuth user={user}><HomePage user={user} /> </RequireAuth>} />
       <Route path={`/${pathPrefix}/data`} element={<RequireAuth user={user}><DataPage user={user} /> </RequireAuth>} />
       <Route path={`/${pathPrefix}/integrations`} element={<RequireAuth user={user}><IntegrationsPage user={user} /> </RequireAuth>} />
       <Route path={`/${pathPrefix}/integration/:id`} element={<RequireAuth user={user}><IntegrationDetailsPage user={user} /> </RequireAuth>} />
       <Route path={`/${pathPrefix}/workflows`} element={<RequireAuth user={user}><WorkflowsPage user={user} /> </RequireAuth>} />
-      <Route path={`/${pathPrefix}/login`} element={ user && user.apiKey ? <Navigate to="/" replace /> : <LoginPage />} />
+      <Route path={`/${pathPrefix}/login`} element={user && user.apiKey ? <Navigate to="/" replace /> : <LoginPage />} />
       <Route path={`/${pathPrefix}/account`} element={<RequireAuth user={user}><AccountPage user={user} /> </RequireAuth>} />
       <Route path={`/${pathPrefix}/workflow/:id`} element={<RequireAuth user={user}><WorkflowPage user={user} /> </RequireAuth>} />
+      <Route path={`/${pathPrefix}/workflow/:workflowId/result/:workflowDagResultId/artifact/:artifactId`} element={<RequireAuth user={user}><ArtifactDetailsPage user={user} /> </RequireAuth>} />
```

Comment:
Not 100% sure about the routing here. Open to feedback and happy to update the route used here.

I do, however, know that we need to have the workflowResultId and artifactResultId on hand to be able to get the data that we need for this Artifact details page.

Technically speaking, this is more of an ArtifactResults detail page.

Reply:
Discussed this route some more with @likawind and outlined some possible solutions here:
https://www.notion.so/aqueducthq/UI-Redesign-API-Routes-ed8b1f5945474683a78c6ae4bf5cfc7e

TLDR: we're going to need the workflowId and workflowDagResultId as part of the route to be able to drill down into the artifact that we want. Whether we implement new routes for fetching the artifact in one go is still TBD. In the interim at least, we can fetch the whole workflow and traverse it's dag to get to the data that we need.